Assets/MixedRealityToolkit.SDK/Experimental/TapToPlace/HideTapToPlaceLabel.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Utilities
+{
+    /// <summary>
+    /// Class to toggle the visibility of a label on a Tap to Place object in the TapToPlaceExample scene.
+    /// </summary>
+    public class HideTapToPlaceLabel : MonoBehaviour
+    {
+        private TapToPlace tapToPlace;
+        private GameObject placeableObjectLabel;
+        void Start()
+        {
+            tapToPlace = gameObject.GetComponent<TapToPlace>();
+            if (tapToPlace != null)
+            {
+                placeableObjectLabel = gameObject.transform.GetChild(0).gameObject;
```

Comment:
It's generally not a good idea to assume a particular child is the gameobject you want. It should either be getcomponent or some reference preset in editor